models/aeif_cond_beta_multisynapse.cpp

```diff
@@ -459,24 +459,31 @@ aeif_cond_beta_multisynapse::calibrate()
     P_.receptor_types_[ i ] = i + 1;
   }
 
-  V_.g0_ex_.resize( P_.num_of_receptors_ );
-  V_.g0_in_.resize( P_.num_of_receptors_ );
+  V_.g0_.resize( P_.num_of_receptors_ );
 
   for ( size_t i = 0; i < P_.num_of_receptors_; ++i )
   {
-    if ( P_.taus_decay[ i ] == P_.taus_rise[ i ] ) // alpha function limit
-    { // use normalization for alpha function in this case
-      V_.g0_ex_[ i ] = V_.g0_in_[ i ] = 1.0 * numerics::e / P_.taus_decay[ i ];
-    }
-    else
+    // denominator is computed here to check that it is != 0
+    double denom1 = P_.taus_decay[ i ] - P_.taus_rise[ i ];
```

Comment:
Your code is correct, but the logic a bit difficult to follow, especially the `if ( denom2 == 0 )` at the end. Since this code is in `calibrate()`, thus rarely executed, I would give legibility precedence over performance and use
```c++
const double denom1 = ...;
const double denom2 = ...;
if ( denom1 == 0 or denom2 == 0 )
{
  // alpha case
}
else
{ 
  // beta case
}